README.md

```diff
@@ -89,6 +89,10 @@ import 'jest-dom/extend-expect'
 import axiosMock from 'axios' // the mock lives in a __mocks__ directory
 import Fetch from '../fetch' // see the tests for a full implementation
 
+// Once we mock the module we can provide a mockResolvedValueOnce for .get 
+// that returns the fake data we want our test to assert against.
+jest.mock('axios');
```

Comment:
This isn't necessary because the comment on line 89 indicates that there's a mock in the `__mocks__` directory.